### PR TITLE
<Description> : Adding UIImageView constructor test.

### PR DESCRIPTION
**Pull request by Khalian**
<Type> : test

## Checklist

- [ ] New Extension
- [x] New Test
- [ ] Changed more than one extension, but all changes are related
- [x] Trivial change (doesn't require changelog)

### CODE REVIEW

File: EZSwiftExtensionsTests/UIImageViewTests.swift [hunk 1]
```diff
@@ -9,8 +9,13 @@
 import XCTest
 
 class UIImageViewTests: XCTestCase {
-
     
-
+    // Tests the UIImageView constructor derived from UIView
+    public func testUIImageConvenienceInit() {
```
Comment (lfarah) [inline EZSwiftExtensionsTests/UIImageViewTests.swift]: I'd change to `testUIImageViewConvenienceInit`

**Comment by Khalian** [comment]
Addressing issue in https://github.com/goktugyil/EZSwiftExtensions/issues/404

File: EZSwiftExtensionsTests/UIImageViewTests.swift (hunk 1 above)
Comment (Khalian) [inline EZSwiftExtensionsTests/UIImageViewTests.swift]: Done

**Comment by Khalian** [comment]
@lfarah I had to reimplement the constructor (extensions do not seem to be inherited). Would you mind taking a look at this again?

**Comment by Khalian** [comment]
I am merging this in, since it is breaking the customer experience.

**Pull request merged**